openlibrary/templates/books/edit/edition.html

```diff
@@ -628,12 +628,58 @@
 
 </fieldset>
 
+$if is_privileged_user:
+    <fieldset class="major">
+        <legend>$_("Web Book Providers")</legend>
+        <div class="formBack">
+            $ provider_index = 0
+            $for i, d in enumerate(book.providers):
+                <fieldset class="minor">
+                    <legend>$ungettext('Provider %(count)s', 'Provider %(count)s', loop.index, count=loop.index)</legend>
+                    <div class="formElement">
+                        <div class="label">
+                            <label for="edition--providers--${i}--url">$_("Book URL")</label>
+                        </div>
+                        <div class="input">
+                            <input name="edition--providers--${i}--url" id="edition--providers--${i}--url" type="url" value="$d.url">
+                        </div>
+                    </div>
+                    <div class="formElement">
+                        <div class="label">
+                            <label for="edition--providers--${i}--access">$_("Access Type")</label>
+                        </div>
+                        <div class="input">
+                            <input name="edition--providers--${i}--access" id="edition--providers--${i}--access" value="$d.access">
+                        </div>
+                    </div>
+                    <div class="formElement">
+                        <div class="label">
+                            <label for="edition--providers--${i}--format">$_("File Format")</label>
+                        </div>
+                        <div class="input">
+                            <input name="edition--providers--${i}--format" id="edition--providers--${i}--format" value="$d.format">
+                        </div>
+                    </div>
+                    <div class="formElement">
+                        <div class="label">
+                            <label for="edition--providers--${i}--provider_name">$_("Provider Name")</label>
+                        </div>
+                        <div class="input">
+                            <input name="edition--providers--${i}--provider_name" id="edition--providers--${i}--provider_name" value="$d.provider_name">
+                        </div>
+                    </div>
+                </fieldset>
+                $ provider_index = provider_index + 1
+            <a href="javascript:;" id="add-new-provider" data-index="$provider_index">Add another provider?</a>
+        </div>
+    </fieldset>
+
 $if ctx.user and ctx.user.is_admin():
     <fieldset class="major adminOnly">
         <legend>$_("Admin Only")</legend>
         <div class="formElement">
             <div class="label">
-                <label for="edition-first_sentence">$_("Additional Metadata")</label>
+                <label for="additional_metadata">$_("Additional Metadata")</label>
```

Comment:
This is a bug-fix that is unrelated to this feature.  If uncorrected, clicking on the "Additional Metadata" label causes the "First sentence" textarea to gain focus.